scooby/report.py

```diff
@@ -34,10 +34,34 @@ def __init__(self):
     def system(self) -> str:
         """Return the system/OS name.
 
-        E.g. ``'Linux'``, ``'Windows'``, or ``'Java'``. An empty string is
+        E.g. ``'Linux (name version)'``, ``'Windows'``, or ``'Darwin'``. An empty string is
         returned if the value cannot be determined.
         """
-        return platform().system()
+        s = platform().system()
+        if s == 'Linux':
+            try:
+                s += (
+                    f' ({platform().freedesktop_os_release()["NAME"]} '
+                    + f'{platform().freedesktop_os_release()["VERSION_ID"]})'
+                )
+            except Exception:
+                pass
+        elif s == 'Windows':
+            try:
+                release, version, csd, ptype = platform().win32_ver()
+                s += f' ({release} {version} {csd} {ptype})'
+            except Exception:
+                pass
```

Comment:
@ThomasMGeo, this is what we're trying to validate on Windows. I suspect some of these return values will be empty strings and I want to determine which few should be prioritized. Perhaps this is fine as is and empty strings will effectively be ignored?

Reply:
![platformplatform](https://github.com/banesullivan/scooby/assets/38228140/cdae3e53-674f-489f-a0d2-962a3f9c1fb7)
This is what I get on my machine.